helm/cloud-provider-vsphere/values.yaml

```diff
@@ -16,7 +16,7 @@ global:
 
 cloud-provider-for-vsphere:
   daemonset:
-    image: gsoci.azurecr.io/giantswarm/cpi-vsphere-manager
+    image: gsoci.azurecr.io/giantswarm/cloud-provider-vsphere
```

Comment:
Upstream migrated registries so I updated retagger for this and use the new image.
As a result we don't need a custom retag as they no longer use the stupid generic name `manager`.
https://github.com/giantswarm/retagger/pull/1022